### PR TITLE
Add setup bootnode page, split the rpc node setup from the wss page and add dbsize reference

### PR DESCRIPTION
**Pull request by stakeworld**
Hi, I have added a bootnode setup page, split the rpc from the maintain-wss page (since it is technically another subject) and made reference to the wss page from the rpc and bootnode page. Also made a reference to my database sizes page on the full node page if thats ok. 

Let me know if you have remarks or comments

### CODE REVIEW

**Review by filippoweb3 — COMMENTED** [review]
thanks @stakeworld! I polished the pages and asked for additional review by @DrW3RK ;)

**Comment by DrW3RK** [comment]
Thanks for the PR @stakeworld Will work on merging this PR early next week!

File: docs/maintain/maintain-rpc.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+Enabling remote access to your validator node should not be necessary and is not suggested, as it
+can often lead to security problems
+
+:::
+
+## Set up a Node
+
+Setting up any Substrate-based node relies on a similar process. For example, by default, they will
+all share the same WebSocket connection at port 9944 on localhost. In this example, we'll set up a
+polkadot sync node on a Debian-flavoured server (such as Ubuntu 22.04). Create a new server on your
```
Comment (profradha) [inline docs/maintain/maintain-rpc.md]:
```suggestion
Polkadot sync node on a Debian-flavoured server (such as Ubuntu 22.04). Create a new server on your
```

File: docs/maintain/maintain-rpc.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+instructions. You can install from the default apt repository or build from scratch. The startup
+options in the setup process provide various settings that can be modified.
+
+A typical setting for an externally accessible polkadot archive RPC node would be:
+
+```config
+polkadot --chain polkadot --name myrpc --state-pruning archive --blocks-pruning archive --ws-max-connections 100 --rpc-cors all --rpc-methods Safe --ws-port 9944
+```
+
+Or for a polkadot pruned RPC node:
```
Comment (profradha) [inline docs/maintain/maintain-rpc.md]:
```suggestion
Or for a Polkadot pruned RPC node:
```

File: docs/maintain/maintain-rpc.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+### Archive Node vs. Pruned Node
+
+A pruned node only keeps a limited number of finalized blocks of the network, not its full history.
+Most frequently required actions can be completed with a pruned node, such as displaying account
+balances, making transfers, setting up session keys, staking, etc. An archive node has the full
+history (database) of the network. It can be queried in various ways, such as providing historical
+information regarding transfers, balance histories, and more advanced queries involving past events.
+
+An archive node requires a lot more disk space. At the start of April 2023, polkadot disk usage was
```
Comment (profradha) [inline docs/maintain/maintain-rpc.md]:
```suggestion
An archive node requires a lot more disk space. At the start of April 2023, Polkadot disk usage was
```

**Review by DrW3RK — APPROVED** [review]
Thank you very much for adding this content to the Wiki!

**Pull request merged**